scout/ext/backend/mongo.py

```diff
@@ -269,6 +271,384 @@ def previous_variant(self, document_id):
+      case.display_name, user.name
+    ))
+    case.assignee = user
+    case.save()
+    self.logger.debug("Case updated")
+    
+    return
+
+  def unassign(self, institute, case, user, link):
+    """
+    Unassign a user from a case.
+    
+    This function will create an Event to log that a person has been unassigned
+    from a case. Also the "assignee" on the case will be updated.
+    
+    Arguments:
+      institute (Institute): A Institute object
+      case (Case): A Case object
+      user (User): A User object (Should this be a user id?)
+      link (str): The url to be used in the event
+    """
+    self.logger.info("Creating event for unassigning {0} from {1}".format(
+      user.display_name, case.display_name
+    ))
+    self.create_event(
+      institute=institute,
+      case=case,
+      user=user,
+      link=link,
+      category='case',
+      verb='unassign',
+      subject=case.display_name
+    )
+    
+    self.logger.info("Updating {0} to be unassigned with {1}".format(
+      case.display_name, user.display_name
+    ))
+    case.assignee = None
+    case.save()
+    self.logger.debug("Case updated")
+    
+    return
+
+  def update_status(self, institute, case, user, status, link):
+    """
+    Update the status of a case.
+    
+    This function will create an Event to log that a user have updated the
+    status of a case. Also the status of the case will be updated.
+    
+    Arguments:
+      institute (Institute): A Institute object
+      case (Case): A Case object
+      user (User): A User object
+      status (str): The new status of the case
+      link (str): The url to be used in the event
+    
+    """
+    
+    self.logger.info("Creating event for assigning {0} to {1}".format(
+      user.display_name, case.display_name
+    ))
+    self.create_event(
+      institute=institute,
+      case=case,
+      user=user,
+      link=link,
+      category='case',
+      verb='status',
+      subject=case.display_name
+    )
+    
+    self.logger.info("Updating {0} to status {1}".format(
+      case.display_name, status
+    ))
+    case.status = status
+    case.save()
+    self.logger.debug("Case updated")
+    
+    return
+  
+  def update_synopsis(self, institute, case, user, link, content=""):
+    """
+    Create an Event for updating the synopsis for a case.
+    
+    This function will create an Event and update the synopsis for a case.
+    
+    Arguments:
+      institute (Institute): A Institute object
+      case (Case): A Case object
+      user (User): A User object
+      link (str): The url to be used in the event
+      content (str): The content for what should be added to the synopsis
+      
+    """
+    self.logger.info("Creating event for updating the synopsis for case {0}".format(
+      case.display_name
+    ))
+    
+    self.create_event(
+      institute=institute,
+      case=case,
+      user=user,
```

Comment:
You might need to trick MongoEngine here by adding an id directly... ``.to_dbref()``